### PR TITLE
Get more space, less useless files

### PR DESCRIPTION
**Pull request by NBGamer99**
Because deleting cache is not enough, I found out some more files to be removed mainly in the code folder, as it can get large pretty quickly and they are safe to delete as long as you don't care about fast startups (from my tests i have loads of extensions and still no noticeable difference in startup times).
> We keep that data around to make subsequent startup faster, it's not needed (and can be deleted)
> Cache and CachedData is a folder from Electron / Chromium. Nothing we control currently, but should be free to be deleted between a restart.

Check from here [issue#76546](https://github.com/microsoft/vscode/issues/76546)

Deleting Crashpad files.
> To make Chromium write crash dumps, no additional work is needed. 

It has to do with Chromium based apps such as Discord and Vscode and the way they store crash dumps in the Crashpad/complete folder

Check from [here](https://www.chromium.org/developers/crash-reports/) 

Basically a 300MB~ of removed useless files.

Also added Slack/Cache

### CODE REVIEW

**Comment by ombhd** [comment]
Thanks @NBGamer99 for your contribution.
